### PR TITLE
feat: add gis dictionary

### PR DESCRIPTION
**Pull request by vineyardbovines**
# Add/Fix Dictionary

Dictionary: GIS / PostGIS

## Description

Added a dictionary for GIS and PostGIS terms

## References

- _Any source references._

## Checklist

- [x] By submitting this pull-request, you agree to follow our [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)
- [x] Verify that the title starts with the correct prefix:
  - `fix:` - for minor changes like adding words or fixing spelling issues.
  - `feat:` - for a significant change like adding a whole new set of words to a dictionary.
  - `feat!:` - for breaking changes, like file format or licensing changes.
  - `chore:` - for changes that do not impact the content of dictionaries.

### CODE REVIEW

**Comment by Jason3S** [comment]
@vineyardbovines,

Thank you. Is there a file type or situation where this dictionary is used? In other words, when should the dictionary be enabled?

File: dictionaries/gis/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# CSpell GIS and PostGIS Dictionary
+
+GIS and PostGIS dictionary for cspell.
```
Comment (ccoVeille) [inline dictionaries/gis/README.md:3]: Hi

I have no idea what is GIS or PoatGIS

so adding a short explanation and links could be interesting

Thanks

**Review by ccoVeille — CHANGES_REQUESTED** [review]

**Comment by vineyardbovines** [comment]
@Jason3S i have an ORM that handles some postgres calls with postgis functionality enabled, that's my current use case- there may be more.

File: dictionaries/gis/README.md (hunk 1 above)
Comment (vineyardbovines) [reply dictionaries/gis/README.md]: I added what GIS stands for and a link to PostGIS.

File: dictionaries/gis/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# CSpell GIS and PostGIS Dictionary
+
+GIS (geographic information system) and [PostGIS](https://postgis.net/) dictionary for cspell.
```
Comment (ccoVeille) [inline dictionaries/gis/README.md:3]:
```suggestion
GIS (Geographic Information System) and [PostGIS](https://postgis.net/) dictionary for cspell.
```

mit: use uppercase first letter for the terms that stand for the acronym

**Pull request merged**